code/controllers/subsystems/ao.dm

```diff
@@ -0,0 +1,31 @@
+SUBSYSTEM_DEF(ao)
+	name = "Ambient Occlusion"
```

Comment:
AO is a native byond feature, why is there a subsystem for it? I don't quite understand the purpose of this

Reply:
> AO is a native byond feature, why is there a subsystem for it? I don't quite understand the purpose of this

Zmimic work with this subsystem

Reply:
right but why is it a subsystem? Can you explain what it does?

Reply:
There's two ways of AO implementation, one based on turf overlays (which is why ao subsystem exists - to control this process) and filter-based AO, which doesn't work with zmimic. If I remember correct, zmimic takes appearance of turf, which contain turf's overlays, so non-filter AO will work with zmimic OK.